examples/profilesvc/transport.go

```diff
@@ -227,63 +225,63 @@ func decodeDeleteAddressRequest(_ context.Context, r *http.Request) (request int
 
 func encodePostProfileRequest(ctx context.Context, req *http.Request, request interface{}) error {
 	// r.Methods("POST").Path("/profiles/")
-	req.Method, req.URL.Path = "POST", url.QueryEscape("/profiles/")
+	req.Method, req.URL.Path = "POST", "/profiles/"
 	return encodeRequest(ctx, req, request)
 }
 
 func encodeGetProfileRequest(ctx context.Context, req *http.Request, request interface{}) error {
 	// r.Methods("GET").Path("/profiles/{id}")
 	r := request.(getProfileRequest)
-	req.Method, req.URL.Path = "GET", url.QueryEscape("/profiles/"+r.ID)
+	req.Method, req.URL.Path = "GET", "/profiles/"+r.ID
 	return encodeRequest(ctx, req, request)
 }
 
 func encodePutProfileRequest(ctx context.Context, req *http.Request, request interface{}) error {
 	// r.Methods("PUT").Path("/profiles/{id}")
 	r := request.(putProfileRequest)
-	req.Method, req.URL.Path = "PUT", url.QueryEscape("/profiles/"+r.ID)
+	req.Method, req.URL.Path = "PUT", "/profiles/"+r.ID
 	return encodeRequest(ctx, req, request)
 }
 
 func encodePatchProfileRequest(ctx context.Context, req *http.Request, request interface{}) error {
 	// r.Methods("PATCH").Path("/profiles/{id}")
 	r := request.(patchProfileRequest)
-	req.Method, req.URL.Path = "PATCH", url.QueryEscape("/profiles/"+r.ID)
+	req.Method, req.URL.Path = "PATCH", "/profiles/"+r.ID
 	return encodeRequest(ctx, req, request)
 }
 
 func encodeDeleteProfileRequest(ctx context.Context, req *http.Request, request interface{}) error {
 	// r.Methods("DELETE").Path("/profiles/{id}")
 	r := request.(deleteProfileRequest)
-	req.Method, req.URL.Path = "DELETE", url.QueryEscape("/profiles/"+r.ID)
+	req.Method, req.URL.Path = "DELETE", "/profiles/"+r.ID
 	return encodeRequest(ctx, req, request)
 }
 
 func encodeGetAddressesRequest(ctx context.Context, req *http.Request, request interface{}) error {
 	// r.Methods("GET").Path("/profiles/{id}/addresses/")
 	r := request.(getAddressesRequest)
-	req.Method, req.URL.Path = "GET", url.QueryEscape("/profiles/"+r.ProfileID+"/addresses/")
+	req.Method, req.URL.Path = "GET", "/profiles/"+r.ProfileID+"/addresses/"
 	return encodeRequest(ctx, req, request)
 }
 
 func encodeGetAddressRequest(ctx context.Context, req *http.Request, request interface{}) error {
 	// r.Methods("GET").Path("/profiles/{id}/addresses/{addressID}")
 	r := request.(getAddressRequest)
-	req.Method, req.URL.Path = "GET", url.QueryEscape("/profiles/"+r.ProfileID+"/addresses/"+r.AddressID)
+	req.Method, req.URL.Path = "GET", "/profiles/"+r.ProfileID+"/addresses/"+r.AddressID
```

Comment:
Likewise the two IDs here? (Other comments elided, but you get the same issue with other methods.)